segment-set-term-title.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+func segmentSetTermTitle(p *powerline) {
+	var setTitle string
+
+	if *p.args.Shell == "bash" {
```

Comment:
We also support other shells than bash or zsh – can you move this part to the other shell-specific variables?